File: BlaBlaCar.BL/ODT/TripModels/SearchTripModel.cs
namespace BlaBlaCar.BL.ODT.TripModels
{
    public class SearchTripModel
    {
        public string StartPlace { get; set; }
        public string EndPlace { get; set; }
        public DateTime StartTime { get; set; }
        public int CountOfSeats { get; set; }
        public int Skip { get; set; } = 0;
        public int Take { get; set; } = int.MaxValue;
    }
}

File: web-client/src/app/interfaces/trip-interfaces/search-trip-model.ts
import { TripOrderBy } from "../../enums/trip-order-by";

export interface SearchTripModel {
    startPlace: string;
    endPlace: string;
    startTime: Date | string;
    countOfSeats: number;
    orderBy?: TripOrderBy;
    take?: number
    skip?: number
}
